lib/config/config.json.sample

```diff
@@ -0,0 +1,33 @@
+{
+	"prod": {
+		"workers": 1,
+		"port": 9124,
+    "hostname": "127.0.0.1",
+		"upstream": "ws://localhost:9125",
+		"retryDelay": 10000,
+		"closeTimer": 15000,
+		"enableInstrumentation": true,
+    "instrumentationTimer": 60000
+	},
+	"test": {
+		"workers": 1,
+		"port": 8999,
+    "hostname": "127.0.0.1",
+		"upstream": "ws://localhost:8999",
+		"retryDelay": 10000,
```

Comment:
The JSON structure has indentation issues it seems...

Reply:
In local machine looks good github is goofing up

Reply:
ah yes. please resolve this comment. Looks good 
![image](https://user-images.githubusercontent.com/22256912/129864402-2bc30c5c-8772-4035-97ab-1cdda6e6fc99.png)